tools/recon/list_interfaces.rb

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env ruby
```

Comment:
I'm just going to delete this tool. It's pretty useless (and it's included in another gem already)

Reply:
Do you want me to make another commit?
Or is this happening in an edit/another PR?

Reply:
If you want to update here, please do. I have another PR to get rid of the network_interfaces gem for most versions of Metasploit, since the functionality is largely built into standard Ruby.

Reply:
I'll take care of it.